### PR TITLE
chore: return 404 for update-my-current-challenge

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: api-server/src/server/boot/settings.js [hunk 1]
```diff
@@ -5,7 +5,10 @@ import isURL from 'validator/lib/isURL';
 
 import { isValidUsername } from '../../../../utils/validate';
 import { alertTypes } from '../../common/utils/flash.js';
-import { deprecatedEndpoint } from '../utils/deprecatedEndpoint';
+import {
+  deprecatedEndpoint,
```
Comment (Sboonny) [inline api-server/src/server/boot/settings.js:9]: I am sorry, but isn't that deleted. shouldn't it be disabled-endpoints

File: api-server/src/server/utils/disabled-endpoints.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  return res.status(410).json({
+    message: {
+      type: 'info',
+      message: 'Please reload the app, this feature is no longer available.'
+    }
+  });
+}
+
+export function temporarilyDisabledEndpoint(_, res) {
+  return res.status(404).json({
```
Comment (ShaunSHamilton) [inline api-server/src/server/utils/disabled-endpoints.js:11]:
```suggestion

  return res.status(410).json({
```

Comment (ShaunSHamilton) [inline api-server/src/server/utils/disabled-endpoints.js:11]:
```suggestion


  return res.status(410).json({
```

**Review by raisedadead — APPROVED** [review]

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Pull request merged**